### ISSUE TITLE
Needs a ELI5 version

### ISSUE THREAD

**Issue by iamcreasy** [alex/what-happens-when#87]
We need an ELI5 version of this document.

ELI5 : Explain Like I am 5

**Comment by TomasHubelbauer** [comment]
I have to disagree on this one, this project states its goal is to not skip out on anything as opposed to numerous online articles with the same subject that only scratch the surface. This is an expansion of ELI5 version easily accessible everywhere and the aim of this project is to be a follow-up to that.

**Comment by alex** [comment]
Yeah, this is out of scope for this project.
